### PR TITLE
Add aliases for paging to bat completer

### PR DESCRIPTION
**Pull request by Lillecarl**
Quite self explanatory 😄

### CODE REVIEW

File: completers/bat_completer/cmd/root.go [hunk 1]
```diff
@@ -37,6 +37,8 @@ func init() {
 	rootCmd.Flags().String("nonprintable-notation", "", "Set notation for non-printable characters")
 	rootCmd.Flags().BoolP("number", "n", false, "Show line numbers")
 	rootCmd.Flags().String("paging", "", "Specify when to use the pager, or use `-P` to disable")
+	rootCmd.Flags().BoolP("no-pager", "", false, "Disable paging")
```
Comment (rsteube) [inline completers/bat_completer/cmd/root.go:40]:
```go
 	rootCmd.Flags().BoolP("list-languages", "L", false, "Display all supported languages")
 	rootCmd.Flags().Bool("list-themes", false, "Display all supported highlighting themes")
 	rootCmd.Flags().StringSliceP("map-syntax", "m", []string{}, "Use the specified syntax for files matching the glob pattern")
+	rootCmd.Flags().BoolP("no-pager", "", false, "Disable paging")
+	rootCmd.Flags().BoolP("no-paging", "", false, "Disable paging")
 	rootCmd.Flags().String("nonprintable-notation", "", "Set notation for non-printable characters")
 	rootCmd.Flags().BoolP("number", "n", false, "Show line numbers")
 	rootCmd.Flags().String("paging", "", "Specify when to use the pager, or use `-P` to disable")
-	rootCmd.Flags().BoolP("no-pager", "", false, "Disable paging")
-	rootCmd.Flags().BoolP("no-paging", "", false, "Disable paging")
 	rootCmd.Flags().CountP("plain", "p", "Show plain style")
 	rootCmd.Flags().BoolP("show-all", "A", false, "Show non-printable characters")
 	rootCmd.Flags().String("style", "", "Comma-separated list of style elements to display")
```

**Review by rsteube — CHANGES_REQUESTED** [review]
diff --git a/completers/bat_completer/cmd/root.go b/completers/bat_completer/cmd/root.go
index 22a892c5..bc63ee1d 100644
--- a/completers/bat_completer/cmd/root.go
+++ b/completers/bat_completer/cmd/root.go
@@ -34,11 +34,11 @@ func init() {
 	rootCmd.Flags().BoolP("list-languages", "L", false, "Display all supported languages")
 	rootCmd.Flags().Bool("list-themes", false, "Display all supported highlighting themes")
 	rootCmd.Flags().StringSliceP("map-syntax", "m", []string{}, "Use the specified syntax for files matching the glob pattern")
+	rootCmd.Flags().BoolP("no-pager", "", false, "Disable paging")
+	rootCmd.Flags().BoolP("no-paging", "", false, "Disable paging")
 	rootCmd.Flags().String("nonprintable-notation", "", "Set notation for non-printable characters")
 	rootCmd.Flags().BoolP("number", "n", false, "Show line numbers")
 	rootCmd.Flags().String("paging", "", "Specify when to use the pager, or use `-P` to disable")
-	rootCmd.Flags().BoolP("no-pager", "", false, "Disable paging")
-	rootCmd.Flags().BoolP("no-paging", "", false, "Disable paging")
 	rootCmd.Flags().CountP("plain", "p", "Show plain style")
 	rootCmd.Flags().BoolP("show-all", "A", false, "Show non-printable characters")
 	rootCmd.Flags().String("style", "", "Comma-separated list of style elements to display")

**Comment by Lillecarl** [comment]
@rsteube Fixed the sorting. Just out of curiosity though, do you have an automated way for these? 😄

**Comment by rsteube** [comment]
linter: https://github.com/rsteube/carapace-bin/tree/master/cmd/carapace-lint
formatter: https://github.com/rsteube/carapace-bin/tree/master/cmd/carapace-fmt

**Pull request merged**